salt/returners/zabbix_return.py

```diff
@@ -54,9 +54,22 @@ def zbx():
         return False
 
 
-def zabbix_send(key, host, output):
-    cmd = zbx()['sender'] + " -c " + zbx()['config'] + " -s " + host + " -k " + key + " -o \"" + output +"\""
-    __salt__['cmd.shell'](cmd)
+def zabbix_send(key, host, output): 
+    f = open('/etc/zabbix/zabbix_agentd.conf','r') 
```

Comment:
We prefer that you use the context manager available at `salt.utils.fopen` instead, please. Also, it looks like another reviewer left a comment on a possible optimization here.